functions/go/apply-setters/applysetters/apply_setters_test.go

```diff
@@ -315,6 +315,66 @@ spec:
 `,
 			errMsg: `invalid setter pattern for array node: "${images}:${tag}"`,
 		},
+		{
+			name: "error: no input",
+			config: `
+data: ~
```

Comment:
Should we use `{}` to indicate an empty map?

Reply:
Done.